metadatastore/commands.py

```diff
@@ -39,167 +99,512 @@ def inner(*args, **kwargs):
     return inner
 
 
-def db_disconnect():
-    """Helper function to deal with stateful connections to mongoengine"""
-    mongoengine.connection.disconnect(ALIAS)
-    for collection in [RunStart, BeamlineConfig, RunStop, EventDescriptor,
-                       Event, DataKey]:
-        collection._collection = None
+def _cache_runstart(rs):
```

Comment:
change this input parameter to `runstart` to be symmetric with the other `_cache_*` functions